test/es5-test.js

```diff
@@ -2,11 +2,11 @@ const assert = require("assert");
 const runLintWithFixtures = require("./lib/runLintWithFixtures");
 
 describe("es5", () => {
-  it("should get expected errors and warninigs with es5 config", () => {
-    const result = runLintWithFixtures("es5");
+  it("should get expected errors and warninigs with es5 config", async () => {
+    const result = await runLintWithFixtures("es5");
     assert.deepStrictEqual(result, {
       "error.js": {
-        errors: ["no-unused-vars", "no-redeclare"]
```

Comment:
The order of the errors is not the same between Node v10 and v12, so I temporarily disable the errors.